server/src/weaverbird/backends/pandas_executor/steps/date_extract.py

```diff
@@ -102,45 +107,45 @@ def execute_date_extract(
             # the result should be returned with 0-ed time information
             result = to_datetime(result.dt.date)
         elif dt_info == "firstDayOfPreviousQuarter":
-            first_month_of_quarter = 3 * ((serie_dt.month - 1) // 3) + 1
+            first_month_of_quarter = 3 * ((series_dt_accessor.month - 1) // 3) + 1
             first_month_of_prev_q = first_month_of_quarter - 3
             first_month_of_prev_q = first_month_of_prev_q.replace({-2: 10})
             result = to_datetime(
                 DataFrame(
                     {
-                        "year": serie_dt.year - (first_month_of_prev_q == 10),
+                        "year": series_dt_accessor.year - (first_month_of_prev_q == 10),
```

Comment:
I don't like the usage of `True == 1 / False == 0` here. It's not golf programming.
Not related to this issue but we should probably rewrite this piece of code to be explicit on the why we have 10, and when we want to do what

Reply:
Agreed, I'll submit a master-only patch for that

Reply:
https://github.com/ToucanToco/weaverbird/pull/1669